CHANGELOG.md

```diff
@@ -7,10 +7,12 @@ Notable changes to this project are documented in this file. The format is based
 Breaking changes:
 
 New features:
+- Export `mapCont` and `withCont` functions (#139 by @JordanMartinez)
```

Comment:
If these were introduced in a separate PR and originally meant to be exported, we should probably link to the PR that actually introduced them I think.

Reply:
Looks like the functions were originally added in #70

Reply:
I've added the PR to the changelog entry. Does that suffice?